opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -123,7 +125,7 @@ namespace Opm {
 
 
         const UnitSystem& m_deckUnitSystem;
-        const ParseContext& m_parseContext;
+        const ParseContext m_parseContext;
```

Comment:
It can, but it's mostly moot because:
* Copying it is rather cheap
* It indicates to both users and compilers that this will in fact be copied and stored
* The copy is only done once because of moves.